nifty-script

```diff
@@ -400,7 +400,9 @@ travis_for_sites() {
 
     echo "travis_fold:start:deploy"
     deploy_args=$(./prepare-deploy ${mode} ${SITE} ${TRAVIS_BUILD_NUMBER})
-    make -C ${SITE} deploy-${mode} ${deploy_args}
+    if [ $mode = "staging" ]; then
```

Comment:
Counterpointer:  I proceeded this way to keep the below production-$site tag stuff in place.  I also felt better with a tighter if than relying on earlier bail-out code, but that's much less of a point than the tag stuff.  Thoughts?